docs/how_to_guides/miscellaneous/how_to_write_a_how_to_guide.rst

```diff
@@ -70,6 +71,40 @@ If the user has data in Mongo and wants to configure a Datasource, no additional
 
 **Additional resources**: Additional resources, usually external (i.e. not within the Great Expectations documentation) and usually shown as a list. To avoid link rot, please use this section sparingly, and prefer links to stable, well-maintained resources.
 
+Code-heavy vs process-heavy guides
+##################################
+
+Broadly speaking, there are two kinds of How-to Guides: code-heavy and process-heavy. All guides are about following a specific sequence of steps. In code-heavy guides, most or all of the steps are expressed in technical syntax: code snippets, JSON or YAML objects, CLI commands, etc. In process-heavy guides, many of the steps are things that must be done manually.
+
+Most guides are code-heavy. Because it's about writing, this guide happens to be process-heavy.
+
+When writing a guide that could go either way, please prefer code-heavy, since they tend to make for better replication.
+
+Indentation, bolding, and code blocks
+#####################################
+
+**For code-heavy guides**
+
+- Treat the first sentence of each step like a header.
+	- Use short, complete, imperative sentences: ("Paste the YAML snippet into your config file", "Run great_expectations init")
+	- Header text should be **bold**.
+	- Avoid links or inline code, since RST files do note support nesting them within bolded text. If your header must include text that would normally be a link or inline code, please repeat it in the body text, and use a link or code block there.
+- Indent content within steps.
+- Any time the user needs to do something, it should be in a code block.
+	- Please follow this convention even if the text in the code block is somewhat redundant against the text of the step. 
+	- Clear, sequential code blocks are easy for the eye to follow. They encourage a health copy-and-modify development pattern.
+- All of these styles are modeled in the :ref:`How-to guide template file <how_to_guides__miscellaneous__how_to_template>`.
+
+**For code-heavy guides**
```

Comment:
I'm assuming this is for process-heavy guides?

Reply:
Yes.

In my defense, this one is still marked "draft".